modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/src/main/java/com/liferay/dynamic/data/mapping/form/web/internal/display/context/util/DDMFormDisplayContextUtil.java

```diff
@@ -40,6 +42,16 @@ public static void addCaptchaDDMFormField(
 
 		captchaDDMFormField.setDataType("string");
 
+		LocalizedValue localizedValue = new LocalizedValue(
+			LocaleUtil.getDefault());
+
+		localizedValue.addString(
+			LocaleUtil.getDefault(), DDMFormFieldTypeConstants.CAPTCHA);
+
+		captchaDDMFormField.setLabel(localizedValue);
+
+		captchaDDMFormField.setShowLabel(false);
+
```

Comment:
@igor-franca please move the 
```
captchaDDMFormField.setShowLabel(false);
```` 
to be after the 

```
captchaDDMFormField.setProperty("url", captchaResourceURL);
``` 
to follow alphabetical order.